app/index.html

```diff
@@ -116,48 +122,52 @@ <h1 class="hero-title">Polymer Project</h1>
+        <a class="onload-fadein product-grid-item left" href="https://lit-element.polymer-project.org" style="animation-delay: .15s; background: #00A3E6; background: linear-gradient(315deg, rgba(255,255,255,0.5) 0%, rgba(255,255,255,0) 100%), linear-gradient(315deg, #3FC7FF 0%, #00A3E6 100%)">
+          <div class="thumb" style="background-image:url(/images/thumbnails/thumb_litElement.svg); padding-top: 39%; margin: 10px 8%;"></div>
           <h2>LitElement</h2>
           <p>A simple base class for creating fast, lightweight web components.</p>
           <span class="status">Prerelease</span>
         </a>
-        <a class="product-grid-item right" href="https://pwa-starter-kit.polymer-project.org" style="background: #6700DF; background: linear-gradient(315deg, rgba(255,255,255,0.5) 0%, rgba(255,255,255,0) 100%), linear-gradient(315deg, #AF6AFF 0%, #6700DF 100%)">
-          <div class="thumb" style="background-image:url(/images/thumbnails/thumb_pwa.svg); padding-top: 48%"></div>
+        <a class="onload-fadein product-grid-item right" href="https://pwa-starter-kit.polymer-project.org" style="animation-delay: .3s; background: #6700DF; background: linear-gradient(315deg, rgba(255,255,255,0.5) 0%, rgba(255,255,255,0) 100%), linear-gradient(315deg, #AF6AFF 0%, #6700DF 100%)">
+          <div class="thumb" style="background-image:url(/images/thumbnails/thumb_pwa.svg); padding-top: 46%; margin: 0px 13%;"></div>
           <h2>PWA Starter Kit</h2>
           <p>Starter templates for building full-featured Progressive Web Apps from web components.</p>
           <span class="status">Prerelease</span>
         </a>
-        <a class="product-grid-item left" href="https://lit-html.polymer-project.org" style="background: #FF2F2F; background: linear-gradient(315deg, rgba(255,255,255,0.5) 0%, rgba(255,255,255,0) 100%), linear-gradient(315deg, #FA5A5A 0%, #FF2F2F 100%)">
+        <a class="onload-fadein product-grid-item left" href="https://github.com/material-components/material-components-web-components" style="animation-delay: .25s; background: #00D67C; background: linear-gradient(315deg, rgba(255,255,255,0.5) 0%, rgba(255,255,255,0) 100%), linear-gradient(315deg, #20FFA7 0%, #00D67C 100%)">
+          <div class="thumb" style="background-image:url(/images/thumbnails/thumb_material.svg); padding-top: 46%; margin: 10px 15%;"></div>
+          <h2>Material Web Components</h2>
+          <p>High-fidelity realizations of Google's Material Design spec that work anywhere on the web.</p>
+          <span class="status">Prerelease</span>
+        </a>
+        <a class="onload-fadein product-grid-item right" href="https://lit-html.polymer-project.org" style="animation-delay: .3s; background: #FF2F2F; background: linear-gradient(315deg, rgba(255,255,255,0.5) 0%, rgba(255,255,255,0) 100%), linear-gradient(315deg, #FA5A5A 0%, #FF2F2F 100%)">
```

Comment:
@yuinchien Just curious if the change in orders ("Material Web Components" before "lit-html") is intentional.